lib/TPP/Transforms/Utils/VNNIUtils.cpp

```diff
@@ -69,6 +69,17 @@ FailureOr<AffineDimExpr> isInVnniLayout(linalg::GenericOp linalgOp,
   return failure();
 }
 
+bool isInVnniLayout(VnniOperandRank expectedRank, VectorType vector) {
+  return isInVnniLayout((int64_t)expectedRank, vector);
```

Comment:
this wrapper seems unnecessary, this C-style cast is not appropriate.

Reply:
Agree, will get rid of it.